<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>🔌 USB 设备管理器 - Windows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="headerWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <pointsize>32</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>🔌 USB 设备管理器</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="userLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>👤 用户: User</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="usbTab">
       <attribute name="title">
        <string>🔌 USB 设备</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="refreshUsbBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>🔄 刷新设备列表</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="usbTable">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>设备名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>制造商</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>序列号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>USB 总线</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>传输速度</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>VID:PID</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="driveTab">
       <attribute name="title">
        <string>💾 U 盘管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QGroupBox" name="drivesGroupBox">
          <property name="title">
           <string>📀 已挂载的 U 盘</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="refreshDriveBtn">
               <property name="text">
                <string>🔄 刷新 U 盘列表</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="drivesTable">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>设备名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>挂载路径</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>文件系统</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>总容量</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>已使用</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>可用空间</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="fileOpsGroupBox">
          <property name="title">
           <string>📝 文件操作</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="filenameLabel">
               <property name="text">
                <string>📄 文件名:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filenameInput">
               <property name="text">
                <string>test.txt</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="textContent">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>120</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>在此输入要写入的文本内容...</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="writeTextBtn">
               <property name="text">
                <string>💾 写入文本文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="uploadFileBtn">
               <property name="text">
                <string>📤 上传文件到 U 盘</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="showHiddenCheck">
               <property name="text">
                <string>👁️ 显示隐藏文件</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="speedLabel">
               <property name="text">
                <string>传输速度: 0 MB/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="filesGroupBox">
          <property name="title">
           <string>📂 文件列表</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableWidget" name="filesTable">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>文件名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>类型</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>大小</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>操作</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
